### PR TITLE
HDDS-3922. SCM : Display the pipeline info on scm web page

### PR DESCRIPTION
**Pull request by smitajoshi12**
## What changes were proposed in this pull request?
Display Pipeline info on SCM Web URL

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-3922

## How was this patch tested?
Manually

image

### CODE REVIEW

**Review by dombizita — COMMENTED** [review]
Thanks for the patch @smitajoshi12, I added some small comments inline.

File: hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm-overview.html [hunk 1]
```diff
@@ -48,6 +48,17 @@ <h2>Node counts</h2>
     </tbody>
 </table>
 
+<h2>PipeLine Info</h2>
```
[the remark this answers is not in the document]
Comment (smitajoshi12) [reply hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm-overview.html]: Done Changes

File: hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm.js [hunk 2]
```diff
@@ -30,6 +30,10 @@
                 .then(function (result) {
                     ctrl.nodemanagermetrics = result.data.beans[0];
                 });
+            $http.get("jmx?qry=Hadoop:service=SCMPipelineManager,name=SCMPipelineManagerInfo")
+            .then(function (result) {
+                ctrl.pieplinesrmetrics = result.data.beans[0];
```
[the remark this answers is not in the document]
Comment (smitajoshi12) [reply hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm.js]: Done Changes

**Comment by smitajoshi12** [comment]
No Need to work on JIRA Nandkishor closed JIRA so closing PR

**Pull request closed without merging**